Modix/Controllers/AutocompleteController.cs

```diff
@@ -46,10 +48,15 @@ public async Task<IActionResult> AutocompleteUsers(string query)
             var result = UserGuild.Users is null
                 ? Enumerable.Empty<ModixUser>()
                 : UserGuild.Users
-                    .Where(d => d.Username.OrdinalContains(query))
+                    .Where(d => d.Username.OrdinalContains(query) || d.Id.ToString() == query)
                     .Take(10)
-                    .Select(d => ModixUser.FromSocketGuildUser(d));
+                    .Select(ModixUser.FromIUser);
 
+            if (!result.Any() && ulong.TryParse(query, out var userId))
+            {
+                var user = await _userService.GetUserAsync(userId);
```

Comment:
This should be `var user = await _userService.GetUserInformationAsync(UserGuild.Id, userId);`